src/layout/index.js

```diff
@@ -1,39 +1,81 @@
+import F from 'futil-js'
 import Awaiter from './Awaiter'
+import BarChart from './BarChart'
+import Checkbox from './Checkbox'
+import CheckButton from './CheckButton'
+import DefaultIcon from '../DefaultIcon'
+import Dynamic from './Dynamic'
+import ExpandableTable from './ExpandableTable'
 import Flex from './Flex'
 import Grid from './Grid'
+import LensInput from './LensInput'
 import Modal from './Modal'
-import Popover from './Popover'
-import SpacedList from './SpacedList'
-import TextHighlight from './TextHighlight'
 import ModalPicker from './ModalPicker'
 import NestedPicker from './NestedPicker'
-import Dynamic from './Dynamic'
-import BarChart from './BarChart'
-import { Tag, TagsInput } from './TagsInput'
-import LensInput from './LensInput'
-import ExpandableTable from './ExpandableTable'
+import Popover from './Popover'
 import Portal from './Portal'
-import CheckButton from './CheckButton'
+import RemoveTagIcon from './RemoveTagIcon'
+import SpacedList from './SpacedList'
 import StepsAccordion, { AccordionStep } from './StepsAccordion'
+import { Tag, TagsInput } from './TagsInput'
+import TextHighlight from './TextHighlight'
 
-export {
+import { defaultTheme } from '../utils/theme'
+
+let theme = {
```

Comment:
I don't think these components should be part of the themeable: Awaiter, Flex, Grid, Dynamic, LensInput, ModalPicker, PopoverContents, Portal, StepsAccordion, TagComponent, Wrap.